lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -165,6 +183,22 @@ def restart_evmserverd
         evmserverd_service = LinuxAdmin::Service.new("evmserverd")
         evmserverd_service.restart if evmserverd_service.running?
       end
+
+      def deactivate
+        @secure = nil
+        @server_host_is_ipaddr = nil
+        configure_messaging_type("miq_queue") # Settings.prototype.messaging_type = 'miq_queue'
+        restart_evmserverd
+        remove_installed_files
+      end
+
+      def secure?
+        @secure ||= server_port == 9_093
+      end
+
+      def server_host_is_ipaddr?
+        @server_host_is_ipaddr ||= server_host.ipaddress?
```

Comment:
Same here with caching a boolean, and also same question if this is even necessary.  If it's not necessary, then I'd argue even further on this one that the method might not be needed at all since the method name is practically identical to the code in question

```ruby
server_host_is_ipaddr?
#vs
server_host.ipaddress?
```